src/app/components/MediaLoader/index.tsx

```diff
@@ -301,10 +304,14 @@ const MediaLoader = ({ blocks, className, embedded, uniqueId }: Props) => {
         )}
         {captionBlock && (
           <Caption
-            className={isPortraitVideo ? 'portrait-caption' : ''}
+            className={isPortrait ? 'portrait-caption' : ''}
             block={captionBlock}
             type={mediaType}
-            css={isPortraitVideo && styles.captionPortrait}
+            css={[
+              styles.figure(embedded),
+              isAudio && styles.captionAudio,
+              !isAudio && [isPortrait && styles.captionPortrait],
```

Comment:
I wonder if this needs to move to within the !isAudio clause? 
```suggestion
              isAudio && styles.captionAudio,
              !isAudio && [styles.figure(embedded), isPortrait && styles.captionPortrait],
```

Reply:
Because the MediaArticlePage is getting this extra padding now, and I am not sure if it's needed or not.

Reply:
I don't think it needs to be moved to work. The [] syntax should combine styles. 

That said - maybe we don't need those styles on audio players - in which case it could be moved. If we'll never have portrait audio then it could be:

isAudio && styles.captionAudio,
!isAudio && styles.figure(embedded),
isPortrait && styles.captionPortrait,

Reply:
Yes, I did think that - although less common, it would still be possible. The example that we're using on the live page happens to have a landscape image, and I imagine this will be true for most audio players. 

I like your suggestion above - I'll update it and see if it works.

Reply:
In fact... `styles.figure(embedded),` looks like an addition to the Caption CSS. I am now wondering if it should be removed or not... because this same CSS is used within the media loader container (placeholder / `figure` tag)...

Reply:
I removed it, and it has not impacted the Visual UX of the Audio Post on Live Page, nor a Media Article Page with Audio Post + Caption.